pkg/api/types.go

```diff
@@ -195,6 +195,119 @@ type VolumeSource struct {
 	NFS *NFSVolumeSource `json:"nfs"`
 }
 
+// Similar to VolumeSource but meant for the administrator who creates PVs.
+// Exactly one of its members must be set.
+type PersistentVolumeSource struct {
+	// GCEPersistentDisk represents a GCE Disk resource that is attached to a
+	// kubelet's host machine and then exposed to the pod.
+	GCEPersistentDisk *GCEPersistentDiskVolumeSource `json:"gcePersistentDisk"`
+	// HostPath represents a directory on the host.
+	// This is useful for development and testing only.
+	// on-host storage is not supported in any way
+	HostPath *HostPathVolumeSource `json:"hostPath"`
+}
+
+type PersistentVolume struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
+
+	//Spec defines a persistent volume owned by the cluster
```

Comment:
There's a general sense that you don't cross namespace boundaries, pretty much ever.  So attaching PVs to a namespace really limits their use (or we break that rule).

There's no concept of a type that exists sometimes in namespaces and sometimes not.  I don't think we want that, but I guess I am open to arguments.  I think it will make @derekwaynecarr cry.

We will eventually have ACLs such that your fancy 1TB volume (just large enough for acceptance testing :) can only be bound to claims by users in your warehouse project.

In short, I think the congruence of this and nodes is powerful and breaking that should be considered precedent setting.  I.e. if we do this, we should consider the same change for nodes.